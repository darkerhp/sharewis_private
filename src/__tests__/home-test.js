'use strict';

import React from 'react';
import { expect } from 'chai';
import { shallow } from 'enzyme';
import { Text } from 'react-native';

const Home = require.requireActual('../home').default;

describe("<Home />", () => {
  it("should contain the welcome text", () => {
    const homeComponent = shallow(<Home />);

    expect(homeComponent
      .find(Text)
      .first()
      .props()
      .children
    ).to.equal("ShareWis ACT");
  });
});
