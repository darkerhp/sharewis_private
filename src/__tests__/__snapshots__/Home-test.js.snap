exports[`Home renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  onLayout={undefined}
  onResponderGrant={[Function bound touchableHandleResponderGrant]}
  onResponderMove={[Function bound touchableHandleResponderMove]}
  onResponderRelease={[Function bound touchableHandleResponderRelease]}
  onResponderTerminate={[Function bound touchableHandleResponderTerminate]}
  onResponderTerminationRequest={[Function bound touchableHandleResponderTerminationRequest]}
  onStartShouldSetResponder={[Function bound touchableHandleStartShouldSetResponder]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent"
      },
      undefined
    ]
  }
  testID={undefined}>
  <Text>
    ShareWis ACT
  </Text>
</View>
`;
